neon_intrinsics/advsimd.md

```diff
@@ -5808,3 +5808,70 @@ The intrinsics in this section are guarded by the macro ``__ARM_NEON``.
+| <code>poly16x8_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vluti2_laneq_p16" target="_blank">vluti2_laneq_p16</a>(<br>&nbsp;&nbsp;&nbsp;&nbsp; poly16x4_t vn,<br>&nbsp;&nbsp;&nbsp;&nbsp; uint8x16_t vm,<br>&nbsp;&nbsp;&nbsp;&nbsp; const int index)</code>         | `vn -> Vn.8H`<br>`vm -> Vm`<br>`0 <= index <= 7`  | `LUTI2 Vd.8H, {Vn.8H}, Vm[index]`   | `Vd.8H -> result`  | `A64`                     |
+| <code>poly16x8_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vluti2q_lane_p16" target="_blank">vluti2q_lane_p16</a>(<br>&nbsp;&nbsp;&nbsp;&nbsp; poly16x8_t vn,<br>&nbsp;&nbsp;&nbsp;&nbsp; uint8x8_t vm,<br>&nbsp;&nbsp;&nbsp;&nbsp; const int index)</code>          | `vn -> Vn.8H`<br>`vm -> Vm`<br>`0 <= index <= 3`  | `LUTI2 Vd.8H, {Vn.8H}, Vm[index]`   | `Vd.8H -> result`  | `A64`                     |
+| <code>poly16x8_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vluti2q_laneq_p16" target="_blank">vluti2q_laneq_p16</a>(<br>&nbsp;&nbsp;&nbsp;&nbsp; poly16x8_t vn,<br>&nbsp;&nbsp;&nbsp;&nbsp; uint8x16_t vm,<br>&nbsp;&nbsp;&nbsp;&nbsp; const int index)</code>       | `vn -> Vn.8H`<br>`vm -> Vm`<br>`0 <= index <= 7`  | `LUTI2 Vd.8H, {Vn.8H}, Vm[index]`   | `Vd.8H -> result`  | `A64`                     |
+
+#### Lookup table read with 4-bit indices
+
+| Intrinsic                                                                                                                                                                                                                                                                                                     | Argument preparation                                                              | AArch64 Instruction                        | Result             | Supported architectures   |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|--------------------------------------------|--------------------|---------------------------|
+| <code>uint8x16_t <a href="https://developer.arm.com/architectures/instruction-sets/intrinsics/vluti4q_lane_u8" target="_blank">vluti4q_lane_u8</a>(<br>&nbsp;&nbsp;&nbsp;&nbsp; uint8x16_t vn,<br>&nbsp;&nbsp;&nbsp;&nbsp; uint8x16_t vm,<br>&nbsp;&nbsp;&nbsp;&nbsp; const int index)</code>                 | `vn -> Vn.16B`<br>`vm -> Vm`<br>`0 <= index <= 1`                                 | `LUTI4 Vd.16B, {Vn.16B}, Vm[index]`        | `Vd.16B -> result` | `A64`                     |
```

Comment:
`vluti4... ` should probably be laneq instead of lane as they use 128bit index vector.

Reply:
Done